Sources/SymDiff/source/RVTCheck.cs

```diff
@@ -480,20 +480,18 @@ public static int RVTRunVerificationTask(SDiff.VerificationTask vt, VC.Condition
 
             vcgen = BoogieVerify.InitializeVC(newProg);
             //SDiff.Boogie.Process.ResolveAndTypeCheck(newProg, "");
-            var newDict = SDiff.Boogie.Process.BuildProgramDictionary(newProg.TopLevelDeclarations.ToList());
             //newEq = (Implementation)newDict.Get(vt.Eq.Name + "$IMPL");
-
+            var newDict = SDiff.Boogie.Process.BuildProgramDictionary(newProg.TopLevelDeclarations.ToList());
```

Comment:
Is this change meaningful?

Reply:
No its not. I was trying to fix something for RVTCheck and this stayed